.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "editor.tabSize": 2,
```

Comment:
I guess it was a mistake, but I don't think we want to enforce this setting (just newlines and trimming whitespace 🙇)